vademecum_VT.md

```diff
@@ -0,0 +1,67 @@
+**_Si consiglia di leggere prima il file README.md_**
+
+# Scopri Firefox e Mozilla!  |VT|
+
+##### Una breve presentazione tecnica dei progetti Mozilla
+
+Capitoli:
+* Presentazione
+* Firefox
+* Test Pilot
+* Altri esperimenti
+* Strumenti utilizzati
+* Unisciti a noi. Diventa un volontario Mozilla Italia ora!
+
+### Presentazione
+**Mozilla Foundation** è un'organizzazione no-profit, presente ovunque nel mondo. La comunità italiana si chiama **Mozilla Italia** (**mozillaitalia.orgs**).
+Questa è la versione tecnica, indirizzata perlopiù a un pubblico con conoscenze più specifiche e specializzanti.
```

Comment:
perlopiù si scrive attaccato?

Reply:
Mi hai fatto venire un dubbio xD http://www.treccani.it/vocabolario/perlopiu/